src/packer.pkr.hcl

```
@@ -136,19 +143,56 @@ build {
   }
 
   provisioner "windows-restart" {
-    # Wait a maximum of 30 minutes for Windows to restart.
-    # The build will fail if the restart process takes longer than 30 minutes.
-    restart_timeout = "30m"
+    # Wait a maximum of 5 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 5 minutes.
+    restart_timeout = "5m"
+  }
+
+  provisioner "powershell" {
+    # Wait 30 seconds before executing the next provisioner.
+    # This gives a grace period between restarting Windows and running the next provisioner.
+    # Disable Windows Defender, enable RDP and set wallpaper.
+    # Install Chocolatey and Boxstarter.
```

This feels kind of choppy as a sort of list so I think I would prefer it to just read as a continuous comment. I also added that we're also configuring RDP instead of just installing it with the changes in this PR.

```suggestion
    # Wait 30 seconds before executing the next provisioner. This gives a grace
    # period between restarting Windows and running the next provisioner.
    # Disable Windows Defender, enable and configure RDP, set the desktop
    # wallpaper, and install Chocolatey and Boxstarter.
```